library/build.gradle

```diff
@@ -72,8 +72,6 @@ dependencies {
     annotationProcessor "androidx.room:room-compiler:$roomVersion"
     api "androidx.work:work-runtime:2.2.0"
 
-    implementation 'com.novoda:merlin:1.2.1'
-    implementation 'com.facebook.stetho:stetho:1.5.1'
```

Comment:
use `debugImplementation` just incase someone is using this?

Reply:
I'm not sure a library can declare and export variant dependencies, it would be up to the client to include `debugImplementation download-manager`

I'd argue that the library itself shouldn't include at all stetho!